runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeSearchPanel.java

```diff
@@ -222,10 +223,12 @@ private void processResult(List<ItemPrice> result, String lookup, boolean exactM
 		SwingUtilities.invokeLater(() ->
 		{
 			int index = 0;
+			int natRune = itemManager.getItemPrice(ItemID.NATURE_RUNE);
 			for (GrandExchangeItems item : itemsList)
 			{
 				GrandExchangeItemPanel panel = new GrandExchangeItemPanel(item.getIcon(), item.getName(),
-					item.getItemId(), item.getGePrice(), item.getHaPrice(), item.getGeItemLimit());
+					item.getItemId(), item.getGePrice(), item.getHaPrice(), item.getGeItemLimit(),
+						natRune);
```

Comment:
This should maintain the indent level of L230.
```suggestion
					natRune);
```

Reply:
Looking at it again, there really isn't a reason to not have it on L230, so I moved it up there.